doc/COOKBOOK-FIXERS.md

```diff
@@ -31,7 +31,7 @@ classes.
 * Forked FriendsOfPHP/PHP-CS-Fixer into your own Github Account.
 * Cloned your forked repository locally.
 * Installed the dependencies of PHP CS Fixer using [Composer](https://getcomposer.org/).
-* You have read [`CONTRIBUTING.md`](CONTRIBUTING.md).
+* You have read [`CONTRIBUTING.md`](/CONTRIBUTING.md).
```

Comment:
Because without that, it's relative path. So it is `/doc/CONTRIBUTING.md`, which is missing.